xlators/mgmt/glusterd/src/glusterd-bitd-svc.c

```diff
@@ -149,9 +148,6 @@ glusterd_bitdsvc_reconfigure()
     glusterd_conf_t *priv = NULL;
     gf_boolean_t identical = _gf_false;
 
-    this = THIS;
```

Comment:
Why we are removing this line?

Reply:
Ack..it shouldn't be removed. Will add it in the next commit..